crates/sui-storage/src/write_ahead_log.rs

```diff
@@ -39,10 +42,20 @@ pub trait TxGuard<'a>: Drop {
 pub trait WriteAheadLog<'a, C> {
```

Comment:
This is a bit of a nit but what we are doing here is really a recovery log. Not sure if the term difference is that significant: https://www.ibm.com/docs/en/tsmfsan/7.1.0?topic=SSSQZW_7.1.0/com.ibm.itsm.srv.doc/c_how_reclog_works.html